examples/vpc_service_control/onprem_project/main.tf

```diff
@@ -0,0 +1,285 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file("${var.service_account_file}")}"
+}
+
+resource "google_project" "on_prem_network_project" {
+  name = "On Prem Network"
+  project_id = "${var.project_id}"
+  org_id     = "${var.organization_id}"
+  billing_account	= "${var.billing_account_id}"
+  auto_create_network = true
+}
+
+
+/* STEP 1A: DELETE THIS COMMENT LINE TO RESERVE STATIC IP (costs ~$0.24/day) 
+resource "google_compute_address" "onprem_vpn_ip" {
+  address_type = "EXTERNAL"
+  name         = "onprem-vpn-ip"
+  network_tier = "PREMIUM"
+  project      = "${google_project.on_prem_network_project.project_id}"
+  region       = "${var.cloud_router_region}"
+} 
+
+output "ip_addr_of_onprem_vpn_router" {
```

Comment:
```suggestion
```

Reply:
Why delete this line?  We need this info.  We take the IP address of the onprem VPN router, and we feed that value into the tfvars file for our VPC SC project.

Reply:
I asked to delete as a follow up to my comment [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/4#discussion_r295592261) outputs should be in outputs.tf file